intelmq/lib/pipeline.py

```diff
@@ -245,7 +245,7 @@ def send(self, message: str, path: str = "_default",
                         "OOM command not allowed when used memory > 'maxmemory'." in exc.args[0]:
                     raise MemoryError(exc.args[0])
                 elif 'Redis is configured to save RDB snapshots, but is currently not able to persist on disk' in exc.args[0]:
-                    raise IOError(28, 'No space left on device or in memory. Redis can\'t save its snapshots. '
+                    raise OSError(28, 'No space left on device or in memory. Redis can\'t save its snapshots. '
```

Comment:
OSError seems to be a generic error, so the upgrade tool changed everything to that :/

Reply:
https://docs.python.org/3/library/exceptions.html?highlight=ioerror#OSError
> Changed in version 3.3: EnvironmentError, IOError, WindowsError, socket.error, select.error and mmap.error have been merged into OSError, and the constructor may return a subclass.